networking/multiple_networks/primary_networks/about-user-defined-networks.adoc

```diff
@@ -18,7 +15,7 @@ User-defined networks provide cluster administrators and users with highly custo
 * Support for the Localnet topology on both primary and secondary networks will be added in a future version of {product-title}.
 ====
 
-Unlike NADs, which are only namespaced scope, UDNs offer administrators the ability to create and define additional networks spanning multiple namespaces at the cluster level by leveraging the `ClusterUserDefinedNetwork` custom resource (CR). UDNs also offer both administrators and users the ability to define additional networks at the namespace level with the `UserDefinedNetwork` CR. 
+Unlike NADs, which are only namespaced scope, UDNs offer administrators the ability to create and define additional networks spanning multiple namespaces at the cluster level by leveraging the `ClusterUserDefinedNetwork` custom resource (CR). UDNs also offer both administrators and users the ability to define additional networks at the namespace level with the `UserDefinedNetwork` CR.
```

Comment:
```suggestion
Unlike NADs, which are only namespaced scope, UDNs offer administrators the ability to create and define additional networks spanning multiple namespaces at the cluster level by leveraging the `ClusterUserDefinedNetwork` custom resource (CR). UDNs also offer both administrators and users the ability to define additional networks at the namespace level with the `UserDefinedNetwork` CR.
```

this sentence is wrong.. NADs can also be used to create and define multiple networks across namespaces

(at first i thought the whitespace was the change but then when I read the sentence looks like it was already present like this?)

Reply:
maybe you meant you'd need multiple NADs one in each namespace for the cluster admin to do that which can be achieved with a single CRD?

Reply:
like if you create a NAD in each namespace and then use the same networkName then they interconnect those namespaces together ... that's how users have been using NADs to connect namespaces together... but that's why I was asking if the meaning here was to say oh you don't need to do that anymore you can use a single CR for the same purpose.. - if so that makes more sense.. but given your response I am not sure if that's what you had meant.

Reply:
Yeah that is what I meant by `namespace scoped`.